generator/nextjs/template/src/pages/credential-issuance.tsx

```diff
@@ -170,102 +134,136 @@ export default function CredentialIssuance({
-          )}
+      );
+    }
+  };
 
-          {!offer && configOptions.length === 0 && (
-            <div className="py-3">Loading configurations...</div>
-          )}
+  const renderVerifiedHolder = (userId: string) => {
+    return (
+      <div className="pb-4">
+        <p className="font-semibold">
+          Verified holder did (From Affinidi Login)
+        </p>
+        <p>{userId}</p>
+      </div>
+    );
+  };
 
-          {!offer && configOptions.length > 0 && (
-            <Select
-              id="configurationIdSelect"
-              label="Configuration"
-              options={configOptions}
-              value={selectedConfig}
-              onChange={handleConfigurationChange}
-            />
-          )}
+  const renderOffer = (offerTorender: OfferPayload) => {
+    return (
+      <div>
+        <Offer offer={offerTorender}></Offer>
+        <Button id="newIssuance" onClick={clearIssuance}>
+          New issuance
+        </Button>
+      </div>
+    );
+  };
 
-          {!offer && selectedConfig && (
+  return (
+    <>
+      <h1 className="text-2xl font-semibold pb-6">Issue Credentials</h1>
+      {renderErrors()}
+      {!hasErrors && (
+        <div>
+          {renderVerifiedHolder(session.userId)}
+          {offer ? (
+            renderOffer(offer)
+          ) : (
             <div>
-              <Select
-                id="claimMode"
-                label="Claim Mode"
-                options={claimModeOptions}
-                value={claimMode}
-                disabled={isFormDisabled}
-                onChange={handleClaimModeChange}
-              />
-              {typeOptions.length === 0 && (
-                <div className="py-3">Loading credential types...</div>
+              {configurations.isPending && (
+                <div className="py-3">Loading configurations...</div>
+              )}
+              {!configurations.isPending && (
+                <Select
+                  id="configurationIdSelect"
+                  label="Configuration"
+                  options={configurations.data || []}
+                  value={selectedConfigId}
+                  onChange={(value) =>
+                    setFormData({
+                      ...formData,
+                      selectedConfigId: value as string,
+                    })
+                  }
+                />
+              )}
+              {selectedConfigId && (
+                <Select
+                  id="claimMode"
+                  label="Claim Mode"
+                  options={claimModeOptions}
+                  value={claimMode}
+                  disabled={isFormDisabled}
+                  onChange={(val) => setClaimMode(val as string)}
+                />
               )}
-              {typeOptions.length > 0 && (
+              {selectedConfigId && credentialTypes.isPending && (
+                <div>Loading credential types...</div>
+              )}
+              {credentialTypes.isSuccess && (
```

Comment:
Please show a message if there are no credential types asking to create a new one

Reply:
Added!